test/Service.FunctionalTests/TraditionalWebApplicationTests.cs

```diff
@@ -2,97 +2,105 @@
-
-            // Redirected to authorize
-            var location = ResponseAssert.IsRedirect(goToAuthorizeResponse);
-            var oidcCookiesComparisonCriteria = CookieComparison.Strict & ~CookieComparison.NameEquals | CookieComparison.NameStartsWith;
-            ResponseAssert.HasCookie(CreateExpectedSetNonceCookie(), goToAuthorizeResponse, oidcCookiesComparisonCriteria);
-            ResponseAssert.HasCookie(CreateExpectedSetCorrelationIdCookie(), goToAuthorizeResponse, oidcCookiesComparisonCriteria);
-            var authorizeParameters = ResponseAssert.LocationHasQueryParameters<OpenIdConnectMessage>(
-                goToAuthorizeResponse,
-                "state");
-
-            // Navigate to authorize
-            var goToLoginResponse = await client.GetAsync(location);
-
-            // Redirected to login
-            location = ResponseAssert.IsRedirect(goToLoginResponse);
-
-            // Navigate to login
-            var goToAuthorizeWithCookie = await client.GetAsync(location);
-
-            // Stamp a login cookie and redirect back to authorize.
-            location = ResponseAssert.IsRedirect(goToAuthorizeWithCookie);
-            ResponseAssert.HasCookie(".AspNetCore.Identity.Application", goToAuthorizeWithCookie, CookieComparison.NameEquals);
-
-            // Navigate to authorize with a login cookie.
-            var goToSignInOidcCallback = await client.GetAsync(location);
-
-            // Stamp an application session cookie and redirect to relying party callback with an authorization code on the query string.
-            location = ResponseAssert.IsRedirect(goToSignInOidcCallback);
-            ResponseAssert.HasCookie("Microsoft.AspNetCore.Identity.Service", goToSignInOidcCallback, CookieComparison.NameEquals);
-            var callBackQueryParameters = ResponseAssert.LocationHasQueryParameters<OpenIdConnectMessage>(goToSignInOidcCallback, "code", ("state", authorizeParameters.State));
-
-            // Navigate to relying party callback.
-            var goToProtectedResource = await client.GetAsync(location);
-
-            // Stamp a session cookie and redirect to the protected resource.
-            location = ResponseAssert.IsRedirect(goToProtectedResource);
-            ResponseAssert.HasCookie(".AspNetCore.Cookies", goToProtectedResource, CookieComparison.NameEquals);
-            ResponseAssert.HasCookie(CreateExpectedSetCorrelationIdCookie(DateTime.Parse("1/1/1970 12:00:00 AM +00:00")), goToProtectedResource, CookieComparison.Delete);
-            ResponseAssert.HasCookie(CreateExpectedSetNonceCookie(DateTime.Parse("1/1/1970 12:00:00 AM +00:00")), goToProtectedResource, CookieComparison.Delete);
-
-            var protectedResourceResponse = await client.GetAsync(location);
-            ResponseAssert.IsOK(protectedResourceResponse);
-            ResponseAssert.IsHtmlDocument(protectedResourceResponse);
+            using (StartLog(out var loggerfactory, minLogLevel: LogLevel.Debug))
```

Comment:
These tests are flaky, so I'm setting the LogLevel to Debug so we can get more information. https://github.com/aspnet/Identity/issues/1648